### PR TITLE
Small updates to framing language

### PR DESCRIPTION
**Pull request by jules32**
Some language to clarify purpose and audience. 

Part of this thinking is around how the Access pages likely need more community-specific detail (NASA exploratory workshops, NOAA a bit longerterm). Alongside https://github.com/Openscapes/openscapes.cloud/issues/10

@ateucher please review to see what you think, and make any further changes and merge; we can move the work forward in further hackdays too.

### CODE REVIEW

File: password-access.qmd [hunk 1]
```diff
@@ -2,6 +2,8 @@
 title: "Password-based Access"
 ---
 
+This page is written primarily for workshop leads (mentors and other partners) who are inviting people to the Hub, as a collaborator for developing workshop materials or as a participant.
```
Comment (ateucher) [inline password-access.qmd:5]: I think in the password-based access page, this isn't quite right. Maybe something like: 
```suggestion
This page is written primarily for workshop leads (mentors and other partners) who will be inviting participants to sign in to the hub on the day of the workshop, without using GitHub for authentication.
```

File: index.qmd [hunk 2]
```diff
@@ -10,7 +10,7 @@ format:
...
+Welcome! [**Openscapes**](https://openscapes.org) supports organizations exploring cloud computing by partnering with [**2i2c**](https://2i2c.org), who design and administer a global network of community hubs. Openscapes supports partners leading workshops to provide friendly first experiences with cloud computing, and this website is both for users to access JupyterHubs and for workshop leads to administer workshops. Work to date has been co-developed with the NASA Openscapes community via the [**Earthdata Cloud Cookbook**](https://nasa-openscapes.github.io/earthdata-cloud-cookbook/).
```
Comment (ateucher) [inline index.qmd:13]: I like this framing a lot.

**Review by ateucher — APPROVED** [review]
Thanks for this Julie! One small thought around the password access page

**Comment by jules32** [comment]
Thanks Andy! Looks good to me!

**Pull request merged**